cmd/release/config/config.go

```diff
@@ -70,6 +88,23 @@ type Rancher struct {
 	Versions map[string]RancherRelease `json:"versions" validate:"dive,omitempty"`
 }
 
+// UI
+type UI struct {
+	Versions map[string]UIRelease `json:"versions" validate:"dive"`
+}
+
+// Dashboard
+type Dashboard struct {
+	Versions           map[string]DashboardRelease `json:"versions" validate:"dive"`
+	RepoOwner          string                      `json:"repo_owner" validate:"required"`
+	RepoName           string                      `json:"repo_name" validate:"required"`
+	UIRepoOwner        string                      `json:"ui_repo_owner" validate:"required"`
+	UIRepoName         string                      `json:"ui_repo_name" validate:"required"`
+	RancherRepoOwner   string                      `json:"rancher_repo_owner" validate:"required"`
+	RancherRepoName    string                      `json:"rancher_repo_name" validate:"required"`
+	RancherUpstreamURL string                      `json:"rancher_upstream_url" validate:"required"`
+}
```

Comment:
I think you should join these two configs into just one and simplify it as much as you can and also remove the dry_run from the config, getting everything ready is a bit too complex it makes it easy to make mistakes.

Reply:
The UI won't be used anymore, removed that struct.

About `dry_run` field, removed the `json:""` _struct tag_, as it will be populated by the `-dry-run` flag. To make the `-dry-run` flag accessible for the whole project will need some kind of rework, as it currently creates a import cycle if we try to implement it.